### PR TITLE
- Fixed typo in the book: ES6 & Beyond # Chapter 2: Syntax

### PR DESCRIPTION
**Pull request by agreg0ire**
Hope I am not mistaken...

### CODE REVIEW

**Comment by getify** [comment]
Either word works fine... but "more" is the original intended meaning.

File: es6 & beyond/ch2.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ This explicitness on your part, which is up to you to maintain with discipline,
 
 #### `let` + `for`
 
-The only exception I'd make to the preference for the *explicit* form of `let` declaration blocking is a `let` that appears in the header of a `for` loop. The reason may seem nuanced, but I believe it to be one of the more important ES6 features.
+The only exception I'd make to the preference for the *explicit* form of `let` declaration blocking is a `let` that appears in the header of a `for` loop. The reason may seem nuanced, but I believe it to be one of the most important ES6 features.
```
Comment (marius-rizac) [inline es6 & beyond/ch2.md]: @getify in the end is your decision, but I would vote for the change that was made. `one of the most important`. I am not a native English speaker but this is how it sounds more natural to me.
Cheers.

**Comment by getify** [comment]
This isn't a question of what sounds right or wrong to native or non-native speakers. It's a question of degrees, and of my original intent thereof. "One of the more important" puts something in a (smaller) group of the items of **higher** importance. "One of the most important" puts something in the (even smaller) group of items of **highest** importance.

Thinking of it like this in terms of scale, maybe it helps:

* all things
* things of more importance
* things of most importance

**Comment by paulirish** [comment]
"more" seems both fitting and correct to me.

The only grammarian's discussion of this I could find: http://www.grammarphobia.com/blog/2009/07/more-than-you-know.html (tl;dr: it's fine either way.)

**Comment by agreg0ire** [comment]
Thanks for explanation to all readers))

**Pull request closed without merging**